<?xml version="1.0"?>
<launch>

  <arg name="sim" default="false" />

  <arg name="robot_ip" unless="$(arg sim)" />
  <arg name="robot_ip" value="0.0.0.0" if="$(arg sim)" />

  <arg name="ns" default="" />
  <arg name="prefix" default="" />

  <!--arg name="max_velocity" default="3.14159" /--> <!-- [rad/s] -->
  <arg name="max_payload" default="6.0" /> <!-- [kg] -->

  <arg name="base_frame" default="$(arg prefix)base" />
  <arg name="tool_frame" default="$(arg prefix)flang" />

  <!-- Start tmr_driver node -->

  <!--TODO: has ns? -->

  <node name="tmr_driver" pkg="tmr_driver" type="tmr_driver" ns ="$(arg ns)/" output="screen">

    <param name="sim" type="bool" value="$(arg sim)" />

    <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />

    <param name="prefix" type="str" value="$(arg prefix)" />

    <!--param name="max_velocity" type="double" value="$(arg max_velocity)" /-->
    <param name="max_payload" type="double" value="$(arg max_payload)" />

    <param name="base_frame" type="str" value="$(arg base_frame)" />
    <param name="tool_frame" type="str" value="$(arg tool_frame)" />

  </node>

</launch>
